controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -296,7 +296,7 @@ func (r *NodeNetworkConfigurationPolicyReconciler) SetupWithManager(mgr ctrl.Man
 
 	allPolicies := handler.ToRequestsFunc(
 		func(handler.MapObject) []reconcile.Request {
-			log := r.Log.WithValues("allPolicies")
+			log := r.Log.WithName("allPolicies")
```

Comment:
Why? Please use the commit message to elaborate whether it is just a side fix you made cause we had issue in logging or whether it is related to "bump nmstate to 0.3"

Reply:
Done